### ISSUE TITLE
Move DockerHub repo to kedacore

### ISSUE THREAD

**Issue by TsuyoshiUshio** [kedacore/keda-scaler-durable-functions#8]
As a developer, I'd like to move the container image namespace from tsuyoshiushio to kedacore, so that people can see the image as official. 

## Success Criteria
- [x] docker image name is changed to kedacore
- [x] All Deployment Script / YAML file is modified and tested. 
- [ ] Create a Release pipeline to publish the docker image

**Comment by tomkerkhove** [comment]
Opened a PR to update the image name, do you have a preference in terms of release pipeline or shall we align with kedacore/keda and use GitHub actions?

**Comment by TsuyoshiUshio** [comment]
Thank you for submitting PR. I've never used GitHub actions, however, I'm fine to move for it.

**Comment by tomkerkhove** [comment]
I don't really have a preference to be honest, I myself use Azure DevOps for most of my projects.

**Comment by tomkerkhove** [comment]
Happy to help where I can

**Comment by TsuyoshiUshio** [comment]
I merged your PR. Let's try GitHub actions this time.  It might be easier for give permission for the contributors.

**Comment by tomkerkhove** [comment]
Happy to help if I can

**Comment by tomkerkhove** [comment]
What process are you looking at for the release?

1. Build image
2. Push image

Or do you have tests to run already?

**Comment by TsuyoshiUshio** [comment]
I appreciate your help! Ideally, If we create a tag, the pipeline works and push the image would be great.   Before pushing, Could you run the C# based test?  You can refer the C# based test on azure-pipeline.yml on this repo.

**Comment by tomkerkhove** [comment]
Sounds good to me, I can take a crack at that.

**Comment by tomkerkhove** [comment]
Do you already have the CI setup for `azure-pipeline.yml` on Azure DevOps? If so, let's just stick with that then.

**Comment by TsuyoshiUshio** [comment]
Yes. https://github.com/kedacore/keda-scaler-durable-functions/blob/master/azure-pipelines.yml this is the one I'm using on current Azure DevOps. You can see the execution in here. https://durabletaskframework.visualstudio.com/Durable%20Task%20Framework%20CI/_build/results?buildId=873&view=results Unfortunately, it is public but under the Microsoft Internal Subscription.

**Comment by tomkerkhove** [comment]
Then I'll draft up a DevOps YAML to align with that!
